### DISCUSSION TITLE
How do I get started with octernship

### DISCUSSION THREAD

**Discussion by DUMBANIKET** [community/community#66075] [GitHub Education] [answered]
I have not done a single octernship though I am eligible for it (I've contributed some open source project) , I have no clue about the process and how to get started with octernship :)

**Comment by arafajs** [comment, marked as answer]
Here are the steps on how to get started with an Octernship:

1. **Check if you are eligible.** To be eligible for an Octernship, you must meet the following criteria:
    * Be at least 18 years old.
    * Be a student at a participating school.
    * Have an active GitHub account.
    * Have contributed to open source projects.
    * Be a resident of one of the countries where Octernships are available.

2. **Sign up for GitHub Global Campus.** GitHub Global Campus is a program that provides students with access to resources and tools to help them learn and grow as developers. To sign up, visit the GitHub Global Campus website and create an account.

3. **Find an Octernship opportunity.** Once you are signed up for GitHub Global Campus, you can browse Octernship opportunities on the GitHub Octernships website. You can search for opportunities by keyword, location, or industry.

4. **Apply for an Octernship.** When you find an Octernship that you are interested in, click the "Apply" button. You will need to complete an application form and submit a resume or CV.

5. **Be patient.** The application process can take a few weeks. If you are selected for an Octernship, you will be notified by email.

Some additional tips:

* Start contributing to open source projects early. This will help you build your skills and experience, and it will also make you more competitive for Octernship opportunities.
* Network with other developers. Attend meetups, conferences, and online forums. This is a great way to learn about new opportunities and to make connections with people who can help you get an Octernship.
* Be proactive. Don't wait for Octernship opportunities to come to you. Do your research and find opportunities that are a good fit for your skills and interests.

**Reply by shivang21007** [reply]
@arafajs there is written that you should be a active contributor . What does it mean? I frequently doing contributed it various repos , does it sufficient?

**Comment by Aradhya-005** [comment]
when will be the octernship start for 2024

**Reply by mahicodes28** [reply]
?????????????
